target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,13 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_usbdev "phone" "Phone" "${board}:white:phone" "1"
```

Comment:
Have a look at the VR2600v GPL tarball for missing key/leds. You should find something similar to this: https://github.com/lede-project/source/pull/414#discussion_r87491038